client/src/styles/Home.styles.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PartialTheme, Theme } from '@fluentui/theme';
+import { CSSProperties } from 'react';
+
+export const fullScreenStyles = {
+  root: {
+    width: '100%',
+    height: '100%'
+  }
+};
+
+export function containerStyles(theme: PartialTheme | Theme | undefined): any {
+  return {
+    root: {
+      maxWidth: '64rem',
+      width: '100%',
+      height: '100%',
+      display: 'flex',
+      justifyContent: 'left', //use vertical and horizontal align instead
+      alignItems: 'center',
+      margin: 'auto',
+      marginTop: '38px',
+      backgroundColor: '#ffffff',
```

Comment:
Since the `containerStyles` takes the `theme,` can we use the colors from here?

Reply:
This color being for the inner container which has to be white, i think that wouldnt work - the theme bg color is grey and wouldnt work here
![image](https://user-images.githubusercontent.com/100614160/202291114-dd8ce694-2536-41f2-91e0-0cf9e9fdd058.png)

Reply:
I'm sorry that it wasn't clear. The `theme` contains some fixed colors such as white. So I meant we could use like `theme.white` but using color code is easier to see as well!

Reply:
Ohhhh... I didnt know that! Thank you Han! :)